stratify/__init__.py

```diff
@@ -1,9 +1,12 @@
 from __future__ import absolute_import
 
-from ._vinterp import (interpolate, interp_schemes, extrap_schemes,
-                       INTERPOLATE_LINEAR, INTERPOLATE_NEAREST,
+from ._vinterp import (interpolate, interp_schemes,  # noqa: F401
+                       extrap_schemes, INTERPOLATE_LINEAR, INTERPOLATE_NEAREST,
                        EXTRAPOLATE_NAN, EXTRAPOLATE_NEAREST,
-                       EXTRAPOLATE_LINEAR, PyFuncExtrapolator, 
+                       EXTRAPOLATE_LINEAR, PyFuncExtrapolator,
                        PyFuncInterpolator)
+from ._bounded_vinterp import (interpolate as  # noqa: F401
+    interpolate_conservative)
+
```

Comment:
Can this be simplified by renaming _bounded_vinterp.interpolate as _bounded_vinterp.interpolate_conservative?